src/App.jsx

```diff
@@ -14,33 +17,164 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'Sort alphabetically';
+const SORT_LENGTH = 'Sort by length';
+const REVERSE = 'Reverse';
+const RESET = 'Reset';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function styleClass(btn) {
+  switch (btn) {
+    case SORT_ALPHABET:
+      return 'is-info';
+    case SORT_LENGTH:
+      return 'is-success';
+    case REVERSE:
+      return 'is-warning';
+    case RESET:
+      return 'is-danger';
+    default:
+      return '';
+  }
+}
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [style, setStyle] = useState({
+    SORT_ALPHABET: false,
+    SORT_LENGTH: false,
+    REVERSE: false,
```

Comment:
```suggestion
    sortType: '',
    isReversed: false,
```
- variables and properties should be in camelCase
- just store sort type, no need to create a property for each type
- boolean variables/properties should start from `is` or `has`